alerting/src/main/kotlin/org/opensearch/alerting/model/MonitorMetadata.kt

```diff
@@ -14,29 +15,40 @@ import org.opensearch.common.xcontent.XContentParser
 import org.opensearch.common.xcontent.XContentParserUtils
 import org.opensearch.commons.alerting.model.Monitor
 import org.opensearch.commons.alerting.util.instant
+import org.opensearch.index.seqno.SequenceNumbers
 import java.io.IOException
 import java.time.Instant
 
 data class MonitorMetadata(
     val id: String,
+    val seqNo: Long = SequenceNumbers.UNASSIGNED_SEQ_NO,
+    val primaryTerm: Long = SequenceNumbers.UNASSIGNED_PRIMARY_TERM,
     val monitorId: String,
     val lastActionExecutionTimes: List<ActionExecutionTime>,
-    val lastRunContext: Map<String, Any>
+    val lastRunContext: Map<String, Any>,
+    // Maps (sourceIndex + monitorId) --> concreteQueryIndex
+    val sourceToQueryIndexMapping: MutableMap<String, String> = mutableMapOf()
```

Comment:
We will need to add the new property, `sourceToQueryIndexMapping`, [here](https://github.com/opensearch-project/alerting/blob/main/core/src/main/resources/mappings/scheduled-jobs.json#L496)

Reply:
done!